src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer_GetDiagnostics.cs

```diff
@@ -312,7 +332,7 @@ public IDECachedDiagnosticGetter(DiagnosticIncrementalAnalyzer owner) : base(own
 
                     foreach (var document in project.Documents)
                     {
-                        await AppendProjectAndDocumentDiagnosticsAsync(state, project, predicate, cancellationToken).ConfigureAwait(false);
+                        await AppendProjectAndDocumentDiagnosticsAsync(state, document, predicate, cancellationToken).ConfigureAwait(false);
```

Comment:
Thanks @heejaechang. My changes aren't in yet. It is likely Manish's change will go in first. So I will merge his changes.